src/cmd/manage_nsfs.js

```diff
@@ -822,6 +839,318 @@ function _validate_access_keys(argv) {
         })) throw_cli_error(ManageCLIError.AccountSecretKeyFlagComplexity);
 
 }
+async function tape_mangement(argv) {
```

Comment:
again, I think the terms in the code should be around the functionality (glacier), and not the implementation (tape)
```suggestion
async function glacier_mangement(argv) {
```